### DISCUSSION TITLE
Things you miss switching to Seelen UI

### DISCUSSION THREAD

**Discussion by Andrej730** [eythaann/Seelen-UI#418] [General]
Have been using Seelen UI for almost a week and wanted to share the list of things I miss from the usual Windows UI:

- No hide all windows button (previously I'd throw the mouse pointer to the right bottom part of the screen and click, now I do Win+M)
- ~~Harder to open task manager (previously I'd open it from context menu on "Start Button", now I do ctrl+shift+esc)~~ can be opened from toolbar context menu.
- No notification icons in the tray
- No notification icons in the task bar
- Can't pin any of the tray icons

Currently the most annoying are the icons since I'm missing the information about pending notifications I used to have. But I've started to pay more attention to general Windows notifications but general notifications are more distracting.

**Comment by IriGitt** [comment]
> Have been using Seelen UI for almost a week and wanted to share the list of things I miss from the usual Windows UI:
> 
>     * No hide all windows button (previously I'd throw the mouse pointer to the right bottom part of the screen and click, now I do Win+M)
> 
>     * Harder to open task manager (previously I'd open it from context menu on "Start Button", now I do ctrl+shift+esc)
> 
>     * No notification icons in the tray
> 
>     * No notification icons in the task bar
> 
>     * Can't pin any of the tray icons
> 
> 
> Currently the most annoying are the icons since I'm missing the information about pending notifications I used to have. But I've started to pay more attention to general Windows notifications but general notifications are more distracting.

Just to address some of your issues.

You can open task manager by right clicking the fancy toolbar. 

There is also a seperate area for notifations in the toolbar that will display a number when a notif is available.

The pinning the background tray icons should be put in as a feature request.

**Comment by MaximumDude** [comment]
Maybe these are being worked on, but I miss:

-  The context menu on the Start Icon
-  The context menus on Date/Time, WiFi, Sound

The Settings Module has Brightness & minimal Audio control, not quite the same thing.
